addons/mail/models/mail_message_link_preview.py

```diff
@@ -0,0 +1,41 @@
+from odoo import fields, models
+from odoo.addons.mail.tools.discuss import Store
+
+
+class MessageMailLinkPreview(models.Model):
+    _name = "mail.message.link.preview"
+    _inherit = ["bus.listener.mixin"]
+    _description = "Link between link previews and messages"
+
+    link_preview_id = fields.Many2one(
+        "mail.link.preview", index=True, required=True, ondelete="cascade"
+    )
+    message_id = fields.Many2one("mail.message", ondelete="cascade")
+    is_hidden = fields.Boolean()
+    author_id = fields.Many2one(related="message_id.author_id")
+    preview = fields.Char(related="message_id.preview")
```

Comment:
`preview` field on `mail.message.link.preview` looks unused?

Reply:
Those related fields are used by the `message_link_preview_list`.